templates/boilerplate/package.json

```diff
@@ -19,38 +19,41 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^14.0.2",
-    "@react-native-async-storage/async-storage": "1.23.1",
+    "@react-native-async-storage/async-storage": "2.1.2",
     "@react-navigation/bottom-tabs": "^6.5.20",
     "@react-navigation/native": "^6.1.10",
     "@react-navigation/native-stack": "^6.9.18",
     "@tanstack/react-query": "^5.32.1",
     "axios": "^1.6.8",
-    "expo": "~51.0.18",
-    "expo-status-bar": "~1.12.1",
-    "msw": "^2.2.14",
-    "react": "18.2.0",
-    "react-native": "0.74.5",
+    "expo": "^53.0.11",
+    "expo-status-bar": "~2.2.3",
+    "react": "19.0.0",
+    "react-native": "0.79.3",
     "react-native-keyboard-aware-scroll-view": "^0.9.5",
-    "react-native-safe-area-context": "4.10.5",
-    "react-native-screens": "3.31.1"
+    "react-native-safe-area-context": "5.4.0",
+    "react-native-screens": "~4.11.1"
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",
     "@testing-library/jest-native": "^5.4.3",
-    "@testing-library/react-native": "^12.4.5",
+    "@testing-library/react-native": "^13.2.0",
     "@thoughtbot/eslint-config": "^1.0.2",
-    "@types/jest": "^29.5.12",
-    "@types/react": "~18.2.73",
-    "@types/react-test-renderer": "^18.0.7",
+    "@types/jest": "^29.5.14",
+    "@types/react": "~19.0.0",
+    "@types/react-test-renderer": "^19.0.0",
     "babel-jest": "^29.7.0",
     "create-belt-app": "^0.7.3",
     "eslint": "^8.56.0",
-    "jest": "^29.3.1",
-    "jest-expo": "~51.0.3",
+    "msw": "^2.2.14",
```

Comment:
Since `msw` is a library that helps us with tests, it should be a part of `devDependencies` and not `dependencies`.